Week02/lesson03/homework/relaxr_starter_code/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+	<meta charset="utf-8">
+	<meta name="description" content="Your description goes here">
+	<meta name="keywords" content="one, two, three">
+	<title>Relaxr Home</title>
+
+	<!-- external CSS links -->
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/style.css">
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,600,800' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+
+	<div id="container">
```

Comment:
Whoa what happened to the indentation here!! Remind me to check your Atom settings to make sure it's using spaces and not tab characters when you hit tab on your keyboard.. I think that might be the issue